### PR TITLE
Comment out login with Facebook which is not working

### LINKED ISSUE #2509: Comment out "login with Facebook" which is not working
The FB login is not working. Also we have not tested and established the flow of how to add FB to an existing account. Please comment out the login with FB functionality for now.

Only allow login with Email for now.

### PR DESCRIPTION
**Pull request by harsh-jindal**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Comment out login with Facebook which is not working

#### Changes proposed in this pull request:

-
-
-

Fixes #2509

### CODE REVIEW

**Comment by kushthedude** [comment]
@harsh-jindal Please remove the merge conflicts .

File: app/templates/components/settings/application-section.hbs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <h4>{{t 'Facebook'}}</h4>
```
Comment (kushthedude) [inline app/templates/components/settings/application-section.hbs]: Try to write `Connect With Facebook` Here

**Comment by pradeepgangwar** [comment]
@harsh-98 Integration tests are failing. Modify them.

**Comment by harsh-jindal** [comment]
@pradeepgangwar I have made the changes please review

**Comment by kushthedude** [comment]
@harsh-jindal please describe in comment why we have commented it?

**Comment by kushthedude** [comment]
@harsh-jindal please describe in comment why we have commented it?

**Pull request merged**